unit-9/exercise-3.js

```diff
@@ -0,0 +1,7 @@
+const inNumber = (int) => {
+  if( int >= 100) {
+    return ' Số nguyên tố lớn hơn 100'
+  }else {
```

Comment:
```suggestion
  if (int >= 100) {
    return true
  } else {
```
Nhìn kỹ cách đặt dấu cách